graphrag_sdk/kg.py

```diff
@@ -66,7 +66,7 @@ def __init__(
         # Load / Save ontology to database
         if ontology is None:
             # load ontology from DB
-            ontology = Ontology.from_graph(ontology_graph)
+            ontology = Ontology.from_schema_graph(ontology_graph)
```

Comment:
What if `ontology_graph` doesn't exists?
Doesn't look like we make sure `ontology_graph` is present in the DB.